$OpenBSD: patch-source_common_lowres_cpp,v 1.1 2019/07/05 14:55:38 kn Exp $

Fix crash with aq-motion when aq-mode is disabled

Index: source/common/lowres.cpp
--- source/common/lowres.cpp.orig
+++ source/common/lowres.cpp
@@ -71,7 +71,7 @@ bool Lowres::create(x265_param* param, PicYuv *origPic
 
     size_t planesize = lumaStride * (lines + 2 * origPic->m_lumaMarginY);
     size_t padoffset = lumaStride * origPic->m_lumaMarginY + origPic->m_lumaMarginX;
-    if (!!param->rc.aqMode || !!param->rc.hevcAq)
+    if (!!param->rc.aqMode || !!param->rc.hevcAq || !!param->bAQMotion)
     {
         CHECKED_MALLOC_ZERO(qpAqOffset, double, cuCountFullRes);
         CHECKED_MALLOC_ZERO(invQscaleFactor, int, cuCountFullRes);
